schema.py

```diff
@@ -273,55 +273,75 @@
                  "audioOutputSource", "videoInputSource", "audioVolume", "mute", "textToSpeech"]
 }
 
+system_settings_schema = {
+    "language": {"type": "string"},
+    "outputResolution": output_resolution_schema,
+    "memc": {"type": "boolean"},
+    "cec": {"type": "boolean"},
+    "lowLatencyMode": {"type": "boolean"},
+    "matchContentFrameRate": {"type": "string"},
+    "hdrOutputMode": {"type": "string"},
+    "pictureMode": {"type": "string"},
+    "audioOutputMode": {"type": "string"},
+    "audioOutputSource": {"type": "string"},
+    "videoInputSource": {"type": "string"},
+    "audioVolume": {"type": "integer"},
+    "mute": {"type": "boolean"},
+    "textToSpeech": {"type": "boolean"},
+}
+
 # Operation: system/settings/get
 # GetSystemSettingsResponse
 get_system_settings_response_schema = {
     "type": "object",
     "properties": {
-        "status": {"type": "integer"},
-        "error": {"type": ["string", "null"]},
-        "language": {"type": "string"},
-        "outputResolution": output_resolution_schema,
-        "memc": {"type": "boolean"},
-        "cec": {"type": "boolean"},
-        "lowLatencyMode": {"type": "boolean"},
-        "matchContentFrameRate": {"type": "string"},
-        "hdrOutputMode": {"type": "string"},
-        "pictureMode": {"type": "string"},
-        "audioOutputMode": {"type": "string"},
-        "audioOutputSource": {"type": "string"},
-        "videoInputSource": {"type": "string"},
-        "audioVolume": {"type": "integer"},
-        "mute": {"type": "boolean"},
-        "textToSpeech": {"type": "boolean"},
+        "status": {"type": "integer"}
     },
-    "required": ["status", "language", "outputResolution", "memc", "cec", "lowLatencyMode",
-                 "matchContentFrameRate", "hdrOutputMode", "pictureMode", "audioOutputMode",
-                 "audioOutputSource", "videoInputSource", "audioVolume", "mute", "textToSpeech"]
+    "required": ["status"],
+
+    "if": {
+        "properties": {
+            "status": { "const": 200 }
+        }
+    },
+    "then": {
+        "properties": system_settings_schema,
+        "required": system_settings_schema.keys()
```

Comment:
This is my mistake here; `dict.keys()` returns a "view object", whereas jsonschema expects an array type as value of `"required"`. The suggestion below fixes this error:
```
Schema error: dict_keys(['language', 'outputResolution', 'memc', 'cec', 'lowLatencyMode', 'matchContentFrameRate', 'hdrOutputMode', 'pictureMode', 'audioOutputMode', 'audioOutputSource', 'videoInputSource', 'audioVolume', 'mute', 'textToSpeech']) is not of type 'array'
```

```suggestion
        "required": list(system_settings_schema.keys())
```